index.html

```diff
@@ -69,81 +69,107 @@ <h2 class="work-title">my recent works</h2>
+            <h3 class="project-title">Project 4</h3>
             <ul class="tag">
               <li>Ruby on rails</li>
               <li>css</li>
               <li>JavScript</li>
               <li>html</li>
             </ul>
-            <button>See Project</button>
+            <button class="button">See Project</button>
           </div>
         </div>
         <div class="work-card">
           <img src="photo/Img Plaholder.png" alt="" class="work-img" />
           <div class="text">
-            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <h3 class="project-title">Project 5</h3>
             <ul class="tag">
               <li>Ruby on rails</li>
               <li>css</li>
               <li>JavScript</li>
               <li>html</li>
             </ul>
-            <button>See Project</button>
+            <button class="button">See Project</button>
           </div>
         </div>
         <div class="work-card">
           <img src="photo/Img Plaholder.png" alt="" class="work-img" />
           <div class="text">
-            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <h3 class="project-title">Project 6</h3>
             <ul class="tag">
               <li>Ruby on rails</li>
               <li>css</li>
               <li>JavScript</li>
               <li>html</li>
             </ul>
-            <button>See Project</button>
+            <button class="button">See Project</button>
           </div>
+        </div>
+      </div>
       </div>
+    </section>
+
+    <section id="detail">
+      <div class="detail">
+        <img src="photo/detail_img.jpg" alt="" id="mobile-img">
+        <img src="photo/Snapshoot Portfolio.png" alt="" id="desktop-img">
+      <i class="fa-solid fa-xmark" id="close"></i>
+      <div class="detail-caption">
+        <h1 class="detail-title">Keeping track of hundreds of components</h1>
+        <ul class="tag">
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li class="desktop-tag">Codekit</li>
+          <li class="desktop-tag">GitHub</li>
+          <li class="desktop-tag">Bootstrap</li>
+        </ul>
+      </div>
+      <p class="detail-text">
+          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea
+      </p>
+      <div class="detail-button">
+        <button>See Live <i class="fa-solid fa-up-right-from-square"></i></button>
+        <button>See Source <i class="fa-brands fa-github"></i></button>
```

Comment:
- [ ] Please note that the JavaScript code should be implemented as an array of objects to store JavaScript objects containing the information for each project. Please note that each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. :+1: 
- [ ] Kindly note that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. Check an example of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96. :pray: 

If you are able to do it well, it will be easy to add projects in the future since you will only need to add it in one place :grinning: